python/Integration/ChimpInsert/main.py

```diff
@@ -0,0 +1,64 @@
+import requests
+import stripe
+import asyncio
+import json
+from scramjet.streams import Stream
+import mailchimp_marketing
+from mailchimp_marketing import Client
+from mailchimp_marketing.api_client import ApiClientError
+
+
+requires = {
+    'requires': 'pipe',
+    'contentType': 'text/plain'
+}
+
+mailchimp = Client()
+
+def get_info(info, given):
+    result = info["members"]
+    for i in range(0, len(result)):
+        if given == result[i]["email_address"]:
+            return result[i]["id"]
+            
+async def insert_info(info):
+	response = mailchimp.ping.get()
+	info = info.split(" ")
```

Comment:
Consider using more informative variable names something like:

`email, name, surname = info.split(" ")`

or 

`email, name, surname, *rest =  info.split(" ")`

if you do not know how many elements you have.